release/simutrans.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    BleachBit cleaner for
+    Simutrans v111 (by Simutrans Team) and compatible versions
+    
+    This CleanerML mark-up is intended for use with the BleachBit program
+    (available at http://bleachbit.sourceforge.net) and was developed by
+    Erratalog blog (http://erratalog.blogspot.com). The mark-up may be
+    modified and/or redistributed so long as the Erratalog blog is cited
+    as a source for the mark-up or derivative works developed from it.
```

Comment:
I agree, and only realized this after saving all your documentation
and reading it at home last night. I re-coded the mark-up for those
two cleaners as well as a number of additional cleaners. All will be
uploaded momentarily. The mark-up will now be identical to those you
released, plus the following three lines of text:
<---...
...your comments...

CleanerML   : Akelpad v4.7.9 (by Kuznetsov and Aleksander)
Contributor   : Erratalog
Web site      : http://erratalog.blogspot.com
-->


On 8/4/15, Andrew Ziem <notifications@github.com> wrote:
>> @@ -0,0 +1,32 @@
>> +<?xml version="1.0" encoding="UTF-8"?>
>> +<!--
>> +
>> +    BleachBit cleaner for
>> +    Simutrans v111 (by Simutrans Team) and compatible versions
>> +
>> +    This CleanerML mark-up is intended for use with the BleachBit
>> program
>> +    (available at http://bleachbit.sourceforge.net) and was developed by
>> +    Erratalog blog (http://erratalog.blogspot.com). The mark-up may be
>> +    modified and/or redistributed so long as the Erratalog blog is cited
>> +    as a source for the mark-up or derivative works developed from it.
>
> I understand you should have credit for this. However, this creates a
> license that I think is not compatible with the GNU General License, which
> is the license for this repository: please see
> https://github.com/az0/cleanerml#license .
>
> You could instead do something like this, and there is no reason anyone
> would really alter it:
>
> ````This cleaner Copyright (c) 2015 by Mike Vainio
> (http://erratalog.blogspot.com)````
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/az0/cleanerml/pull/94/files#r36260356